### DISCUSSION TITLE
Error connecting to API on Dashboard.

### DISCUSSION THREAD

**Discussion by JRafael91** [supabase/supabase#989] [Questions] [answered]
![image](https://user-images.githubusercontent.com/39107286/113051580-fc3ef280-915a-11eb-95a4-dc1da1aa0163.png)

![image](https://user-images.githubusercontent.com/39107286/113051734-2a243700-915b-11eb-9fba-0f1f65517706.png)

**Comment by inian** [comment, marked as answer]
Sorry for the late reply! Can you send the project ref to beta@supabase.io? I will take a look.
